src/server/controllers/tasks.js

```diff
@@ -52,4 +52,49 @@ export class TaskManager {
       }
     });
   }
+
+
+  /**
+     *
+     *
+     * @param {*} [option={}]
+     * @memberof TaskManager
+     */
+  updateATask() {
+    Task.findById(this.req.params.id).exec((error, tasks) => {
+      if (error) {
+        this.res.json({
+          error: error.message,
+        });
+      } else {
+        tasks.name = this.req.body.name || tasks.name;
+        tasks.body = this.req.body.body || tasks.body;
+        tasks.start_date = this.req.body.start_date || tasks.start_date;
```

Comment:
Use camel case for object property names.

Reply:
ok will change that